.github/renovate.json5

```diff
@@ -295,6 +295,53 @@
         "Dockerfile.clang-format"
       ],
       "enabled": false
+    },
+    {
+      // do not update dependencies from the replace directive from cilium/cilium
+      // see go.mod second replace directive comments for more info
+      "enabled": false,
+      "matchPackageNames": [
+        // All of these packages are maintained on a Cilium fork. Thus, we don't
+        // want to update them automatically.
+        "go.universe.tf/metallb",
+        "github.com/cilium/metallb",
+        "github.com/vishvananda/netlink",
+        "github.com/cilium/netlink",
+        // metallb is still using an old version of "github.com/mdlayher/arp"
+        "github.com/mdlayher/arp",
+        "github.com/miekg/dns",
+        "github.com/cilium/dns",
+        "sigs.k8s.io/controller-tools",
+        "github.com/cilium/controller-tools",
+        "github.com/cilium/client-go",
+      ]
+    },
+    {
+      // not an ignore but almost, please automate this if you have the energy
+      // require manual steps for minor and major cilium/cilium updates
+      "matchPackageNames": [
+        "github.com/cilium/cilium",
+      ],
+      "matchUpdateTypes": [
+        "major",
+        "minor",
+      ],
+      "draftPR": true,
+      "prBodyNotes": [
+        ":warning: Manual steps need to be performed (explaining why this PR is only a draft). First replace directives must be synchronized with cilium/cilium, see the `go.mod` and `pkg/k8s/go.mod` for more info. Then dependencies ignore in `.github/renovate.json5` for the replace directive must be updated accordingly."
```

Comment:
IIUC it also requires updating k8s dependencies. Should renovate mention that in the draft PR too?

Reply:
So this should be automatic if you go get cilium or if you bump and go mod tidy. But maybe I should mention it indeed.